companion/src/simulation/telemetrysimu.cpp

```diff
@@ -154,6 +156,55 @@ void TelemetrySimulator::onLogTimerEvent()
   logPlayback->stepForward(false);
 }
 
+void TelemetrySimulator::onGpsRunLoop()
+{
+    int a = ui -> gps_latlon -> text().contains(",");
```

Comment:
Indentation 2 spaces and not tabs and no spaces around pointers

Reply:
So it should be +  int a = ui->gps_latlon->text().contains(",");
Is that correct?

Reply:
Yes

Reply:
The 2 spaces indentation at each level is throughout the PR. You can see I have asked some other devs about a style guide. With one in place you can configure your IDE of choice to default to the style.

Reply:
still some more pointer refs to be fixed